pywebdriver/plugins/win32print_driver.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 Akretion (https://www.akretion.com).
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import base64
+import tempfile
+import os
+
+from flask import request, jsonify, make_response
+
+from pywebdriver import app, drivers
+from .base_driver import AbstractDriver
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import win32ui, win32print, win32api
+except ImportError:
+    installed = False
+    print("WIN32: win32print python library not installed")
+else:
+
+    class Win32printDriver(AbstractDriver):
+        def get_vendor_product(self):
+            return "cups-icon"
+
+        def get_status(self, **params):
+            messages = []
+            for printer in win32print.EnumPrinters(
+                win32print.PRINTER_ENUM_NAME, None, 5
+            ):
+                messages.append(printer["pPrinterName"])
+            state = {
+                "status": "connected",
+                "messages": messages,
+            }
+            return state
+
+        def printData(self, printer, data, title="Pywebdriver", options=None):
+            if printer.startswith("zebra"):
```

Comment:
Maybe better to use the raw option in options

Reply:
will support also copies options see available options here : https://github.com/akretion/ak-odoo-incubator/blob/10.0/proxy_action/models/helper.py#L27